hyperspy/drawing/utils.py

```diff
@@ -1190,12 +1194,18 @@ def plot_spectra(
         For a list, if its length is less than the number of spectra to plot,
         the colors will be cycled. If `None`, use default matplotlib color
         cycle.
-    line_style: {None, matplotlib line style, list of line_styles}, optional
+    line_style : {None, matplotlib line style, list of line_styles}, optional
         Sets the line style of the plots (no action on 'heatmap').
-        The main line style are '-','--','steps','-.',':'.
+        The main line style are ``'-'``, ``'--'``, ``'-.'``, ``':'``.
         For a list, if its length is less than the number of
         spectra to plot, line_style will be cycled. If
-        If `None`, use continuous lines, eg: ('-','--','steps','-.',':').
+        If `None`, use continuous lines, eg: ``'-'``, ``'--'``, ``'-.'``,
+        ``':'``.
+    drawstyle : {'default', 'steps', 'steps-pre', 'steps-mid', 'steps-post'},
+        default: 'default'
+        The drawstyle determines how the points are connected, no action with
+        ``style='heatmap'``. See matplotlib drawstyle of
+        :py:func:`matplotlib.lines.Line2D` for more information.
     padding : float, optional, default 1.0
```

Comment:
What is the correct style for default value? With or without ':' - this one and the parameter before are inconsistent.

Reply:
This is defined in matplotlib, I have tweaked the docstring to make it more clear. Even if this is not obvious from the names,  [`drawstyle`](https://matplotlib.org/stable/api/_as_gen/matplotlib.lines.Line2D.html#matplotlib.lines.Line2D.set_drawstyle) and [`linestyle`](https://matplotlib.org/stable/api/_as_gen/matplotlib.lines.Line2D.html#matplotlib.lines.Line2D.set_linestyle) are two different parameters.
I am wondering, if we should rename `line_style` to `linestyle` to better match with matplotlib [`linestyle`](https://matplotlib.org/stable/api/_as_gen/matplotlib.lines.Line2D.html#matplotlib.lines.Line2D.set_linestyle)?

Reply:
Using the same terminology as matplotlib makes a lot of sense, I would say. So +1  for renaming.

Reply:
Yes, you are right: there was actually a bug and I simplify a bit the code too. It should be better now!